views/index.dt

```diff
@@ -0,0 +1,21 @@
+!!! 5
+head
+  link(rel="stylesheet", href="css/style.min.css")
+body(style="text-align: center")
+  h2(style="margin-top: 6rem") Dlang-Bot
+  div
+    img(src="img/dlang_bot.jpg")
```

Comment:
maybe add sth. like  `style="max-height: 50%`

Otherwise the dlang-bot is quite big. Here's how it currently looks on my laptop (without modifications) on my Laptop. My screen ends there :/

![image](https://cloud.githubusercontent.com/assets/4370550/17463555/8f41360a-5cc8-11e6-9f10-aaf94ee33306.png)